mountpoint-s3-client/src/s3_crt_client/put_object.rs

```diff
@@ -81,6 +81,7 @@ impl S3CrtClient {
                 span,
                 move |metrics| {
                     if metrics.request_type() == RequestType::CreateMultipartUpload && !metrics.error().is_err() {
+                        // Send signal on a successful CreateMultipartUpload request
```

Comment:
Added a comment here because I do not believe it is clearer to collapse these three conditions into one line.